printer_mods/darkestbo/4 channel relay mount/README.md

```diff
@@ -0,0 +1,14 @@
+4 Channel relay mount for 2020 aluminium profile
+-
+
+This mod is meant to be used with an elegoo (or similar should work with X66 and Y45 interaxis of holes) 4 channel relay
+
+You'll need 
+
+- 2 M5x8 Bottom head screws
+- 2 Tnut / hammerhead nuts
+- 4 M2x10 self screwing screws for plastic
+
+Relay: https://www.amazon.it/AZDelivery-Modulo-Channel-Optoisolatore-Arduino/dp/B078Q8S9S9/ref=sr_1_5_sspa?__mk_it_IT=%C3%85M%C3%85%C5%BD%C3%95%C3%91&dchild=1&keywords=rele+raspbery+elegoo&qid=1611957209&sr=8-5-spons&psc=1&smid=A1X7QLRQH87QA3&spLa=ZW5jcnlwdGVkUXVhbGlmaWVyPUExRDlYN1ZPN1k3TERJJmVuY3J5cHRlZElkPUEwNzQ5MjM1MjRSTEFDQ1MyVFBaViZlbmNyeXB0ZWRBZElkPUEwNDcxMjMzOFJRREMxQVBCVFFBJndpZGdldE5hbWU9c3BfbXRmJmFjdGlvbj1jbGlja1JlZGlyZWN0JmRvTm90TG9nQ2xpY2s9dHJ1ZQ==
```

Comment:
I would shorten that link to just `https://www.amazon.it/AZDelivery-Modulo-Channel-Optoisolatore-Arduino/dp/B078Q8S9S9` .